database/seeds/CentreUsersSeeder.php

```diff
@@ -18,10 +18,21 @@ public function run()
             "password" => bcrypt('store_pass'),
             "role" => "centre_user",
         ]);
+
+        // Attach an initial centre
+        $user1->centres()->attach([
+            1 => ['homeCentre' => true]
+        ]);
+
+        // Get the first centre's sponsor and make two more centres with the same sponsor
+        $sponsor_id = $user1->centres()->first()->sponsor()->first()->id;
```

Comment:
`$sponsor_id = $user1->centres()->first()`... 

- Up to this point because of lazy loading we *may* be dealing with calls to a database, depending on what state $user1 was in. `first()` is like `get()`, but limited to a single record. That gets turned into a single `Centre` model object.
- A `Centre` only has 1 sponsor, so you don't need ...`sponsor()->first()->id`; you can get away with `sponsor->id` as `sponsor` is a property of the model, and `id` is a property of that.